cmd/janitor/gcp_janitor.py

```diff
@@ -318,6 +325,10 @@ def collect(project, zones, age, resource, filt, clear_all):
                 # subcommands want just the name, not the full URL, so strip it.
                 colname = item[resource.condition].rsplit('/', 1)[-1]
                 log('looking for items in %s=%s' % (resource.condition, colname))
+            elif resource.group == 'filestore':
```

Comment:
I'm not familiar with `filestore` so not entirely sure how it works. Were you able to run this script against your own project to make sure this does not break?

Reply:
Yes, I have verified it with my project having filestore instances.